data/campaigns/fri03.wmf/scripting/init.lua

```diff
@@ -1,7 +1,7 @@
 -- =======================================================================
 --                         Frisian Campaign Mission 3
 -- =======================================================================
-set_textdomain("scenario_fri03.wmf")
+push_textdomain("scenario_fri03.wmf")
```

Comment:
do we need to pop the textdomain at the end?

Reply:
None of the scenarios pop their global textdomain. Technically, popping is only needed to ensure that a script which is included from another script cleans up after itself so the calling script is using the same textdomain before and after the include; but a scenario's `init.lua` is the toplevel script, so it does not need to terminate cleanly.

Reply:
ah thanks i am still learning things